example/src/AssertLocation/AssertLocation.tsx

```diff
@@ -67,8 +68,10 @@ const AssertLocation = () => {
     getHotspotDetails(gatewayAddress)
       .then(setHotspot)
       .catch((e) => console.log(e))
-    Onboarding.getOnboardingRecord(gatewayAddress)
-      .then(setOnboardingRecord)
+
+    new OnboardingClient()
```

Comment:
Would it be better to create a global instance somewhere? Or is it intended to make a new instance each time we interact with the `OnboardingClient `?

Reply:
Ideally makers would create a global instance, but for the sake of a tiny demo I think this is fine